db/seeds.rb

```diff
@@ -0,0 +1,11 @@
+messages = [
+  "Welcome to our app!",
+  "It is a pleasure having you in here.",
+  "Thank you for joining us.",
+  "Welcome to the community.",
+  "We hope you feel at home."
+]
+
+messages.each do |content|
+  Message.create(content: content)
+end
```

Comment:
- [ ] To ensure a better and more valid review from your side, please make sure to create and run the migration file that's supposed to have been created with the Message model, this is also to make sure there's a schema (database) in which the messages will be created and fetched from. 🙏